swarm/pss/pss_test.go

```diff
@@ -457,6 +457,76 @@ func TestKeys(t *testing.T) {
 	}
 }
 
+func TestGetPublickeyEntries(t *testing.T) {
```

Comment:
why do we need a simulation to test this simple API call? unit test on pssapi like all other api methods ?

Reply:
changed @zelig